backend/de.metas.picking.rest-api/src/main/java/de/metas/picking/config/MobileUIPickingUserProfile.java

```diff
@@ -33,10 +34,47 @@
 @Builder(toBuilder = true)
 public class MobileUIPickingUserProfile
 {
-	public static final MobileUIPickingUserProfile DEFAULT = builder().name("default").build();
+	public static final MobileUIPickingUserProfile DEFAULT = builder()
+			.name("default")
+			.availablePickingFilters(ImmutableSet.of(PickingJobFilterOption.CUSTOMER, PickingJobFilterOption.DELIVERY_DATE))
```

Comment:
AFAIK in vanilla we don't have filters... so availablePickingFilters shall be empty